mmdet/datasets/pipelines/test_aug.py

```diff
@@ -7,23 +9,35 @@
 @PIPELINES.register_module()
 class MultiScaleFlipAug(object):
 
-    def __init__(self, transforms, img_scale, flip=False):
+    def __init__(self,
+                 transforms,
+                 img_scale,
+                 flip=False,
+                 flip_direction='horizontal'):
         self.transforms = Compose(transforms)
         self.img_scale = img_scale if isinstance(img_scale,
                                                  list) else [img_scale]
         assert mmcv.is_list_of(self.img_scale, tuple)
         self.flip = flip
+        self.flip_direction = flip_direction if isinstance(
+            flip_direction, list) else [flip_direction]
+        assert mmcv.is_list_of(self.flip_direction, str)
+        if not self.flip and len(self.flip_direction) > 1:
```

Comment:
Change from `len(self.flip_direction) > 1` to `self.flip_direction != ['horizontal']`?